cmd/kubeapps-apis/proto/kubeappsapis/plugins/helm/packages/v1alpha1/helm.proto

```diff
@@ -22,7 +22,7 @@ service HelmPackagesService {
     };
   }
 
-  // GetAvailablePackageVersions returns the package versions managed by the 'kapp_controller' plugin
+  // GetAvailablePackageVersions returns the package versions managed by the 'helm' plugin
   rpc GetAvailablePackageVersions(kubeappsapis.core.packages.v1alpha1.GetAvailablePackageVersionsRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackageVersionsResponse) {
     option (google.api.http) = {
       get: "/plugins/fluxv2/packages/v1alpha1/availablepackageversions"
```

Comment:
oops, `fluxv2` instead of `helm`. It also happens in the kapp proto, I've just noticed.

```suggestion
      get: "/plugins/helm/packages/v1alpha1/availablepackageversions"
```